scripts/deploy_assisted_service.sh

```diff
@@ -50,7 +51,7 @@ elif [ "${DEPLOY_TARGET}" == "ocp" ]; then
 else
     print_log "Updating assisted_service params"
     skipper run discovery-infra/update_assisted_service_cm.py ENABLE_AUTH=${ENABLE_AUTH}
-    skipper run "make -C assisted-service/ deploy-all" ${SKIPPER_PARAMS} DEPLOY_TAG=${DEPLOY_TAG} DEPLOY_MANIFEST_PATH=${DEPLOY_MANIFEST_PATH} DEPLOY_MANIFEST_TAG=${DEPLOY_MANIFEST_TAG} NAMESPACE=${NAMESPACE} ENABLE_AUTH=${ENABLE_AUTH} PROFILE=${PROFILE}
+    cd assisted-service/ && skipper run "make deploy-all" ${SKIPPER_PARAMS} ENABLE_KUBE_API=${ENABLE_KUBE_API} DEPLOY_TAG=${DEPLOY_TAG} DEPLOY_MANIFEST_PATH=${DEPLOY_MANIFEST_PATH} DEPLOY_MANIFEST_TAG=${DEPLOY_MANIFEST_TAG} NAMESPACE=${NAMESPACE} ENABLE_AUTH=${ENABLE_AUTH} PROFILE=${PROFILE}
```

Comment:
why do you cd to the directory and not running `skipper run "make -C assisted-service/ deploy-all"` ?

Reply:
The reason I did it because it currently runs with a test-infra skipper image.
If we want to have the dependencies to generate manifests we need to use an assisted-service build image.
I think regardless of this specific need that this would be more appropriate to use an assisted-service build image to deploy assisted-service.

Reply:
Is it because of the controller-gen? 
@tsorya is there a specific reason why assisted build image wasn't used to begin with?

Reply:
yes, the controller gen is the main reason

Reply:
sounds logical i must say but it will require to create 2 build images in every deployment and will take very long time

Reply:
Then to make it work with single build container we need to install some things from this commit https://github.com/openshift/assisted-service/commit/697a281f3a06b8314d7ed74e9a8985d6e142b8f3
Do you think it will better?